test/index.js

```diff
@@ -132,6 +137,7 @@ Swarm.init({
     assert.ok(t1 === t2 && t2 === t3, `Tasks: NoStackingTask concurrent stacking: ${t1} === ${t2} === ${t3}`);
   });
 }).then(_ => { // Tasks: ParallelTask
+  console.log('Tasks: ParallelTask');
```

Comment:
will `console.log` create too many logs? maybe `debug()` would be better since we can enable logs by set env DEBUG=node-swarm if needed?